easybuild/easyconfigs/f/fineRADstructure/fineRADstructure-20170825-goolf-1.7.20_795698d.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'ConfigureMake'
+
+name = "fineRADstructure"
+version = "20170825"
+git_commit = "795698d"
+versionsuffix = "_%s" % (git_commit)
+
+homepage = 'http://cichlid.gurdon.cam.ac.uk/fineRADstructure.html'
+description = """A package for population structure inference from RAD-seq data"""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+source_urls = ['https://github.com/millanek/fineRADstructure/archive/']
+sources = ['%s.tar.gz' % (git_commit)]
```

Comment:
@pescobar Please rename the source on download, see easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#alternative-formats-for-sources